### PR TITLE
Cosmos DB - POST operation for retrieving read only keys

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by sujitvk1 — APPROVED** [review]

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json [hunk 1]
```diff
@@ -433,6 +433,35 @@
...
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/DatabaseAccountListReadOnlyKeysResult"
```
Comment (anuchandy) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: It seems both `DatabaseAccounts_GetReadOnlyKeys` and `DatabaseAccounts_ListReadOnlyKeys` returns the same result. Are we planning to deprecate one of this eventually?

Comment (dmakwana) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: Yes the GET is to be deprecated

Comment (anuchandy) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: thank you. Is this new POST action to retrieve the keys a long running operation?

Comment (dmakwana) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: No

Comment (anuchandy) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: @dmakwana any reason why we use POST for listing keys?

Adding Gaurav - @ravbhatnagar  any concern here from ARM side? the context is:

1. There is already an operation to retrieve the database keys which is based on HTTP GET
2. We are introducing a new operation for retrieving the same database keys which is based on HTTP POST (this is not a long running operation)
3. The operation based on GET will be deprecated.

Comment (ravbhatnagar) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: @anuchandy - Yes, this is the right way. secrets like keys should not be exposed via  a GET. Right way is through a POST. Thanks for checking!

Comment (anuchandy) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json]: Thanks @ravbhatnagar. 

@dmakwana merging the PR.

**Review by anuchandy — APPROVED** [review]

**Pull request merged**